src/js/Templates/Shared/Image.js

```diff
@@ -129,6 +129,11 @@ class Image extends React.Component {
         else if (!prevState.refreshed) {
             newState.error = false
         }
+        // If there was an error loading image but image path has changed,
+        // remove error flag
+        if (prevState.error && prevState.error !== nextProps.image) {
+            newState.error = false;
```

Comment:
```suggestion
            newState.errorPath = null;
```
Just so we're not mixing data types for this value, perhaps we should rename it

Reply:
Good idea, I updated all uses of state.error in https://github.com/smartdevicelink/generic_hmi/pull/373/commits/aaea83e1eb0cda14677c3366d3cb3e73ffee3543